### PR TITLE
Fix MacOS Packaging

### PR DESCRIPTION
**Pull request by DavidMansolino**
Fixes issue #1677.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
✔️: trvis_release_only

File: src/packaging/webots_distro.c [hunk 1]
```diff
@@ -836,8 +836,6 @@ static void create_file(const char *name, int m) {
       fprintf(fd, "cd \"%s/%s/lib/webots\"\n", distribution_path, bundle_name);
       fprintf(fd, "ln -s libssl.1.0.0.dylib libssl.dylib\n");
       fprintf(fd, "ln -s libcrypto.1.0.0.dylib libcrypto.dylib\n");
-      fprintf(fd, "ln -s libssh.4.dylib libssh.dylib\n");
-      fprintf(fd, "ln -s libzip.2.dylib libzip.dylib\n");
```
Comment (omichel) [inline src/packaging/webots_distro.c:840]: Don't we need these libraries for the Darwin-OP robot window?

Comment (omichel) [reply src/packaging/webots_distro.c:840]: Oh yes, I see, they are installed locally...

Comment (DavidMansolino) [reply src/packaging/webots_distro.c:840]: Exactly, actually here this command was just creating a symlink to a file not existing 😄

**Pull request merged**